internal/cmd/operator-sdk/cleanup/cmd.go

```diff
@@ -15,24 +15,43 @@
 package cleanup
 
 import (
+	"context"
+	"fmt"
+	"os"
+	"time"
+
 	"github.com/spf13/cobra"
 
-	"github.com/operator-framework/operator-sdk/internal/cmd/operator-sdk/cleanup/packagemanifests"
+	"github.com/operator-framework/operator-sdk/internal/operator"
 )
 
 func NewCmd() *cobra.Command {
+	var timeout time.Duration
+	cfg := &operator.Configuration{}
+	u := operator.NewUninstall(cfg)
```

Comment:
Nit: move this into the `Run` field's func for clarity.

Reply:
Sounds good. If we add flags to the Uninstaller, we'll have to pull it back out, but I agree this is more clear for how it works now.